definitions/npm/lodash_v4.x.x/flow_v0.104.x-/lodash_v4.x.x.js

```diff
@@ -167,9 +167,12 @@ declare module "lodash" {
     ...
   };
 
+  declare type Key = string | number;
```

Comment:
It looks like object key cannot be anything else: http://www.ecma-international.org/ecma-262/#sec-object-initializer

Can add `| symbol` for Symbol, but it looks like Flow does not support it atm: https://github.com/facebook/flow/issues/3258.